bit-block-boom/2019/_index.es.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Bit Block Boom 2019
```

Comment:
I think this is the same issue as #152. We need an `.es` index page in the parent directory.

Reply:
I have created the file in the main directory.